upup/pkg/fi/cloudup/subnets_test.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Would love to see some negative test cases. If I am reading this correctly we do not have any, could be wrong